igor-web/src/main/java/com/netflix/spinnaker/igor/config/CiConfig.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.config;
+
+import com.netflix.spinnaker.igor.ci.CiBuildService;
+import com.netflix.spinnaker.igor.ci.NoopCiService;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+
+@ConditionalOnProperty("services.ci.enabled")
```

Comment:
similarly, i think you can delete this class in igor since it's not doing anything

Reply:
My understanding is that if you have `services.ci.enabled`, but you're not also supplying a `CiBuildService`, then when you try to use the controller it'll throw the exception.

Because we have the `Optional`,  I don't think we need the conditional on missing bean here.